server/jvm/src/main/scala/org/http4s/server/middleware/CSRF.scala

```diff
@@ -526,11 +526,8 @@ object CSRF {
     )
 
   /** Generate an unsigned CSRF token from a `SecureRandom` */
-  private[middleware] def genTokenString: String = {
-    val bytes = new Array[Byte](CSRFTokenLength)
-    CachedRandom.nextBytes(bytes)
-    encodeHexString(bytes)
-  }
+  private[middleware] def genTokenString: SyncIO[String] =
```

Comment:
I wonder, can we move to `F[_]: Sync` instead of `SyncIO`? It seems `genTokenString` is used only when `Sync` is in-game.

Reply:
Ah, yes I think so :) let's do that 👍